### ISSUE TITLE
Can't setup integration / can't login

### ISSUE THREAD

**Issue by ps-tfriedl** [custom-components/alexa_media_player#846]
**Describe the bug**
My integration stopped working and I was getting a notification to verify my Alexa account in the sidebar - after trying it a couple of times (entering the one time code I got from Amazon) I gave up and and uninstalled and re-installed the integration.

Now I'm trying to login again but I'm stuck at the screen that says "Alexa Media Player - Action Required" - I get the email and SMS from Amazon and I confirm the login but the "Submit" button the dialog does nothing. I just has a short loading indicator and the dialog stays the same. I don't see anything in the logs either (debug activated) - or at least I'm not sure what to look for.

**To Reproduce**
1. Go to Integrations and add the Alexa Media Player integration
2. Fill out details
3. After the dialog with the title "Alexa Media Player - Action Required" appears open the email from Amazon and confirm the login
4. Click Submit in the dialog - nothing happens

**Expected behavior**
After step 4 I expect to have the integration connected to my account.

**System details**
 - Home-assistant (version): 0.112.4
 - Hassio (Yes/No): (Please note you may have to restart hassio 2-3 times to load the latest version of alexapy after an update. This looks like a HA bug).
 - alexa_media  (version from `const.py` or HA startup): 2.10
 - alexapy (version from `pip show alexapy` or HA startup): 1.12

**Comment by alandtse** [comment]
This is a duplicate of #807.  Please see that thread for workaround and request for testers for a solution.
